lib/vernier/output/firefox.rb

```diff
@@ -385,6 +385,25 @@ def string_table
 
         private
 
+        def pretty_name(name)
+          if name.empty?
+            begin
+              tr = ObjectSpace._id2ref(@ruby_thread_id)
+              name = tr.inspect if tr
+            rescue RangeError
+              # Thread was already GC'd
+            end
+          end
+          return name unless name.start_with?("#<Thread")
+          pretty = []
+          obj_address = name[/Thread:(0x\w+)/,1]
+          best_id = name[/\#<Thread:0x\w+@?\s?(.*)\s+\S+>/,1] || ""
+          Gem.path.each { |gem_dir| best_id = best_id.gsub(gem_dir, "...") }
```

Comment:
not totally necessary I guess, but otherwise the paths can end up being pretty huge and we probably only care about the portion of the path after the gem actually starts.